### PR TITLE
Fixes

### LINKED ISSUE #9: newGlyph should return the glyph.
Make newGlyph return the newly created glyph.

### PR DESCRIPTION
**Pull request by jamesgk**
Closes #9, #30

### CODE REVIEW

File: Lib/defcon/objects/layer.py [hunk 1]
```diff
@@ -206,6 +206,7 @@ def newGlyph(self, name):
         glyph.enableNotifications()
         self.postNotification("Layer.GlyphAdded", data=(dict(name=name)))
         self.dirty = True
+        return glyph
```
Comment (anthrotype) [inline Lib/defcon/objects/layer.py]: This should be added to the Font object as well, no?

File: Lib/defcon/objects/glyph.py [hunk 2]
```diff
@@ -722,19 +719,17 @@ def insertAnchor(self, index, anchor):
         This will post a *Glyph.Changed* notification.
         """
         try:
-            assert anchor.glyph != self
+            assert anchor.glyph is None
```
Comment (anthrotype) [inline Lib/defcon/objects/glyph.py]: Whay's the purpose of writing an assertion and then passing if it fails?...

Comment (anthrotype) [inline Lib/defcon/objects/glyph.py]: Oh it passes on `AttributeError`, of course!
Sorry about that. I need a coffee ;)

File: Lib/defcon/objects/layer.py (hunk 1 above)
Comment (jamesgk) [inline Lib/defcon/objects/layer.py]: It was already there: https://github.com/typesupply/defcon/blob/ufo3/Lib/defcon/objects/font.py#L187

**Pull request merged**